src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -50,6 +50,8 @@
  *  Now it returns empty iterable but should return list of repo languages.
  *  Uncomment test RtRepoTest.iteratesLanguages() when done.
  *  See https://developer.github.com/v3/repos/#list-languages for API details.
+ *  @todo #955 Uncomment test RtRepoTest.iteratesLanguages()
```

Comment:
Should only be one space between the `*` and `@todo`. The indentation of the second line is correct. See the existing puzzle above.